### DISCUSSION TITLE
More examples of usage?

### DISCUSSION THREAD

**Discussion by Almenon** [reposaur/reposaur#42] [General]
It would be nice if there were more examples of usage. There's a lot of directions you could take this, so I'm curious what would be some recommended policies.

**Comment by crqrdotcom** [comment]
Hey @Almenon 👋🏽 - thanks for the feedback and sorry for the delay had this lost in my inbox.

Currently, we're using policies for several things such as:

- Check branch protection rules are in place and follow best practices
- Check if required files are present (i.e. code owners, readme, contributing, etc)
- Check if repositories have migrated CI/CD pipelines
- Check if pulls follow conventional commits guidelines, have assignees, labels, body, etc

The possibilities are really _infinite_ and most probably depend from organization to organization.

---

With this said, I've been thinking a lot about the direction I want for the project for next year. One of the things that have been on the paper for a long time is:

1) provide a default policy library with best practices that organizations can use or extend from
2) support for Open Policy Registry (see https://www.openpolicyregistry.io/)

---

Really appreciate any feedback I can get! If you or anyone wants to chat, please [join us on our Discord](https://discord.gg/jpx4sqkQYY) or drop me an email (visible in my profile). I'm more than happy to jump on a call and help people get on board and understand the frustrations they face.

Thanks!
